### ISSUE TITLE
Ghost Overlay Crashes when I try to open Pursuits, but works fine anywhere else

### ISSUE THREAD

**Issue by CWSurpris3** [joshhunt/GhostOverlay#47]
I recently tried using Ghost Overlay when Lightfall dropped. I've clean installed multiple times, restarted my computer, unlinked and relinked my account, and yet nothing has fixed it. If I'm quick enough, I can swap the tab to track a mission on the triumphs and insights tabs, but pursuits specifically keeps crashing game bar as well. 

Any tips or workarounds would be appreciated.

**Comment by KnightWriter147** [comment]
> 

Same Here.  I've been using Ghost Overley for years.  Everything was fine, even the night before Lightfall's release. After the DLC came out, this widget stopped functioning.  I am encountering the same issues stated above.

**Comment by Mvolk89** [comment]
Having the same issue. It also will not load the page to track things.

**Comment by nuttertkk** [comment]
Same here.. Same issue as above.

Also I can't haven't been able to track Catalysts or Craftable weapons, since Witch Queens Launch.

**Comment by CWSurpris3** [comment]
@KnightWriter147 @Marokazam @nuttertkk What tabs are y'all able to see when you open the settings? I only see Pursuits, Insights, and Triumphs, there was one time I was able to see Seasonal Challenges but haven't had it since.

**Comment by nuttertkk** [comment]
> @KnightWriter147 @Marokazam @nuttertkk What tabs are y'all able to see when you open the settings? I only see Pursuits, Insights, and Triumphs, there was one time I was able to see Seasonal Challenges but haven't had it since.

I see "Change Character", Pursuits, Triumphs and Insights, and the Setting Cog across the top bar. See attach image:
![2023-03-07 (2)](https://user-images.githubusercontent.com/6377214/223440276-6c663b4a-720f-453a-9a58-2fb0df5e4004.png)

**Comment by KnightWriter147** [comment]
> @Marokazam @nuttertkk, @CWSurpris3 
I also see the "Change Character" Tab, Pursuits, Triumphs, and Insights, in that order at the top left.  Then the Settings cog at the top right.  
The snippet below is what I see both opening the overlay and after pushing the "Restart" button when I select the "Pursuits" Tab. The "Change Character" tab and Triumphs tab work for me. I have yet to try the Insights tab in-game this season.

![image](https://user-images.githubusercontent.com/86389135/223882186-989ee860-f44e-4db0-9643-7088a9b19445.png)

**Comment by KnightWriter147** [comment]
@CWSurpris3 have you been able to look into the issue?

**Comment by CWSurpris3** [comment]
@KnightWriter147 I have not. Nothing from the creator either. I personally have no idea how to look into this issue specifically... haha the most programming/systems experience I've had is personal dabbling in C++ and a CS college course semester. It is unfortunately just a waiting game...

**Comment by joshhunt** [comment]
This is an annoying one - Bungie stopped publishing one of the database tables used to show Bounties in Lightfall, so the app crashes immediately when trying to show them.

I'm working on a fix that I hope to get out "soon".

**Comment by Mvolk89** [comment]
> This is an annoying one - Bungie stopped publishing one of the database tables used to show Bounties in Lightfall, so the app crashes immediately when trying to show them.
> 
> I'm working on a fix that I hope to get out "soon".

Thanks for updating us!

**Comment by CWSurpris3** [comment]
> This is an annoying one - Bungie stopped publishing one of the database tables used to show Bounties in Lightfall, so the app crashes immediately when trying to show them.
> 
> I'm working on a fix that I hope to get out "soon".

Very appreciated! Good luck on working on the fix, I wish you less bugs and more progress :)

**Comment by joshhunt** [comment]
Have a fix, and on the plus side the Seasons and current release quest categories should update in lock step with the game now (rather than me lazily not updating it manually each season)

![image](https://user-images.githubusercontent.com/46142/228079906-a4860060-8259-4e17-90ef-e28e818818a3.png)

I'll aim to submit to MS Store some time this week, but it's been a while since I've submitted an update so who knows how it'll go 👍🏻

**Comment by arcomador** [comment]
> Have a fix, and on the plus side the Seasons and current release quest categories should update in lock step with the game now (rather than me lazily not updating it manually each season)
> 
> ![image](https://user-images.githubusercontent.com/46142/228079906-a4860060-8259-4e17-90ef-e28e818818a3.png)
> 
> I'll aim to submit to MS Store some time this week, but it's been a while since I've submitted an update so who knows how it'll go 👍🏻

You sir are awesome!

**Comment by exproject** [comment]
> 

Appreciated!

**Comment by theredkoolaid** [comment]
Thanks for the update.

**Comment by CWSurpris3** [comment]
> i have issue recently i tried to use it i cant add or remove anything any time i try it ends up crashing says something isn't playing nice and i either have to have it retry and it keeps crashing everytime i restart it on my pc which is all i have please help fyi i have language set to English US if you need to know

You're having the same issue as the rest of us. Just wait till Josh gets an update on the Microsoft store.

**Comment by Urim512** [comment]
> > i have issue recently i tried to use it i cant add or remove anything any time i try it ends up crashing says something isn't playing nice and i either have to have it retry and it keeps crashing everytime i restart it on my pc which is all i have please help fyi i have language set to English US if you need to know
> 
> You're having the same issue as the rest of us. Just wait till Josh gets an update on the Microsoft store.

i thought it was recently updated?

**Comment by CWSurpris3** [comment]
Thank you very much @joshhunt! No problems since the update. I do have a quick question though: where can I find where to track my catalysts? I'm not seeing anywhere in the triumphs screen...

**Comment by Urim512** [comment]
> Thank you very much @joshhunt! No problems since the update. I do have a quick question though: where can I find where to track my catalysts? I'm not seeing anywhere in the triumphs screen...

i dont think that is a feature for this are you possibly confusing it with the exotic quests for either the weapons or to get the catalyst?

**Comment by Urim512** [comment]
> > Thank you very much @joshhunt! No problems since the update. I do have a quick question though: where can I find where to track my catalysts? I'm not seeing anywhere in the triumphs screen...
> 
> i dont think that is a feature for this are you possibly confusing it with the exotic quests for either the weapons or to get the catalyst?



> Thank you very much @joshhunt! No problems since the update. I do have a quick question though: where can I find where to track my catalysts? I'm not seeing anywhere in the triumphs screen...

i will admit though it would be a nice feature to have on it and have something to track that progress with other than my little light app with my phone hey @joshhunt any chance you can add in next update or patch?

**Comment by CWSurpris3** [comment]
> > Thank you very much @joshhunt! No problems since the update. I do have a quick question though: where can I find where to track my catalysts? I'm not seeing anywhere in the triumphs screen...
> 
> i dont think that is a feature for this are you possibly confusing it with the exotic quests for either the weapons or to get the catalyst?

It certainly was a feature and was something I used Ghost primarily for. You were able to track the progress of how many kills you have left for you to masterwork the catalyst. It's specifically under the Triumphs, it shows you a % of how many kills are needed for the catalyst to be complete, and then it gives you the specific number you have completed out of how many kills total the catalyst will need. 
Whoopsies. In that reddit page, they were talking about the new triumph tracking implemented in destiny 2 itself. Ghost overlay used to have this feature, I assume josh may have dropped it after it was implemented into d2? That's prolly what happened.

Here's what it says on the Ghost overlay website 
![image](https://user-images.githubusercontent.com/82538129/230441264-aef7177f-654a-4b27-aec6-7af27da2adac.png)
